service-commands/src/commands/service-commands.json

```diff
@@ -48,6 +48,9 @@
     ]
   },
  "solana-validator": {
+   "protocol": "http",
+   "host": "localhost",
+   "port": 8899,
```

Comment:
wonder if we could just include an optional "healthCheckEndpoint" here so we don't have to special case it elsewhere